### ISSUE TITLE
mouse.h only supported on 32u4 controllers, Due and Zero

### ISSUE THREAD

**Issue by mvdmolen** [nshadov/screensaver-mouse-jiggler#1]
According to the library mouse.h only works with 32u4 controllers, Due and Zero boards. So Nano does not work. 

I am new to Arduino and could not make screensave-mouse-jiggler work on a Nano ATmega328P.

**Comment by andyman198** [comment]
Yeah I purchased 3 nanos and uploaded the library and found the same thing. Wasted a little money there.

**Comment by Ozzy33** [comment]
Yep, It only seems to work with the 32u4 mpu (arduino micro and Leonardo).
@ nshadov, need to update your docs.
thanks for the code!

**Comment by frijsdijk** [comment]
List of boards where Mouse.h is available:

https://www.arduino.cc/reference/en/language/functions/usb/mouse/
